Updated README.md

ghatwala opened the pull request:
Added the step to increase the per-thread stack size in `conf/jvm.options` to ensure that we don't observe JVM crashes on POWER machine.

ghatwala committed:
Updated README.md

Added the step to increase the per-thread stack size in `conf/jvm.options` to ensure that we don't observe JVM crashes on POWER machine.

odaira commented:
I summarized the necessary steps to run Cassandra on ppc64le in this Wiki page https://github.com/odaira/cassandra/wiki/Running-Apache-Cassandra-on-ppc64le. Shouldn't we add links, rather than describing the duplicate information in README.md?

ghatwala committed:
Updates README.md

Removed the duplicate information pertaining to running Apache Cassandra on ppc64le . Instead added the wiki page link as suggested by review comments in PR#5.

ghatwala commented:
@odaira - sorry about not catching this . I agree on this . Added the wiki link to README.md in https://github.com/hhorii/capi-rowcache/pull/5/commits/2e5a109c77e635e07a6038292412496fc087b11a . Please kindly review.
